### PR TITLE
Fix various misspellings and grammatical errors

### PR DESCRIPTION
**Pull request by Veritius**
## About the PR 
Fixes various misspellings and grammatical errors. Also added contractions for ease of reading.

**Screenshots**

**Changelog**

:cl:
- fix: Fixed various misspellings and grammatical errors

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   name: pickaxe
   parent: BaseItem
   id: Pickaxe
-  description: Notched to perfection, for jebbing it into rocks
+  description: Notched to perfection, for jamming it into rocks
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml:5]:
```suggestion
  description: Notched to perfection, for jabbing it into rocks.
```

File: Resources/Prototypes/Entities/Objects/Weapons/security.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@
   name: flash
   parent: BaseItem
   id: Flash
-  description: An ultrabright flashbulb with a trigger, causes the victum to be dazed and lose their eyesight for a moment. Useless when burnt out.
+  description: An ultrabright flashbulb with a trigger, causes the victim to be dazed and lose their eyesight for a moment. Useless when burnt out.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Weapons/security.yml:35]:
```suggestion
  description: An ultrabright flashbulb with a trigger, which causes the victim to be dazed and lose their eyesight for a moment. Useless when burnt out.
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml (hunk 1 above)
Comment (ElectroJr) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml]: Jeb was also a minecraft dev, so I assume that "jebbing" was meant to be an intentional reference

Comment (Veritius) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml]: Now that I think of it, "jamming" might be better, as the word "jabbing" is more for spears and the use thereof.

**Review by ElectroJr — COMMENTED** [review]
Apparently an empty file was accidentally added ([Resources/Server Info/.fuse_hidden000076f000000001](https://github.com/space-wizards/space-station-14/pull/7397/files#diff-8003d7762004ec156559af4d32b35d039778326809398ce4b0ac1e945626899f))

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**